src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -22,6 +22,7 @@
 
   <PropertyGroup>
     <TestWithCore Condition="'$(TestWithCore)' == '' And '$(TestTFM)' != 'netcoreapp1.0'">false</TestWithCore>
+    <TestAppX Condition="'$(TestWithCore)' == 'false' And '$(TestTFM)' == 'netcore50'">true</TestAppX>
```

Comment:
This also needs to be changed to prevent the native compilation run of these runs from using the same runner.